pyproject.toml

```diff
@@ -43,6 +43,14 @@ opentelemetry-exporter-jaeger-thrift = "^1.13.0"
 optional = true
 dependencies = { sentry-sdk = "^1.11.0" }
 
+[tool.poetry.group.flask_api]
```

Comment:
You'll need to update `.github/workflows/ci.yml` to add `--with fast_api,flask_api` as args to `poetry install`